docker/synapse3_10.Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG SYTEST_IMAGE_TAG=bookworm
```

Comment:
ARG defines an argument that can be set at build time with a parameter to `docker build` (see https://docs.docker.com/engine/reference/builder/#arg). As things currently stand, `SYTEST_IMAGE_TAG` is unused in your dockerfile, so this is rendundant.

That said, I don't think we *need* to hardcode the base image on line 2: currently the only sensible value is `bookworm`, but that might change in the future.